pom.xml

```diff
@@ -131,9 +130,6 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>${maven.compiler.version}</version>
-          <configuration>
-            <release>8</release>
```

Comment:
done by the property now and by the way this was overriding any configuration made by `source/target` parameter.